src/core/load_balancing/xds/xds_wrr_locality.cc

```diff
@@ -167,15 +167,15 @@ absl::Status XdsWrrLocalityLb::UpdateLocked(UpdateArgs args) {
   }
   auto config = args.config.TakeAsSubclass<XdsWrrLocalityLbConfig>();
   // Scan the addresses to find the weight for each locality.
-  std::map<std::string, uint32_t> locality_weights;
+  std::map<RefCountedStringValue, uint32_t> locality_weights;
```

Comment:
Does this needs to be `std::map<RefCountedStringValue, uint32_t, RefCountedStringValueLessThan>`?

Reply:
Haven't gotten a complaint from the compiler, maybe because I'm just comparing it with other `RefCountedStringValue`. We can add it if needed.

Reply:
We define [`operator<(RefCountedStringValue, RefCountredStringValue)`](https://github.com/grpc/grpc/blob/ddb785674ca6c21f9aeee4018b9ef4594ed41cc4/src/core/lib/gprpp/ref_counted_string.h#L126), so it shouldn't be necessary to use `RefCountedStringValueLessThan` unless we need heterogeneous lookups.